packages/react-select/src/components/Option.js

```diff
@@ -31,8 +31,8 @@ export type OptionProps = PropsWithStyles &
     innerProps: InnerProps,
     /* Text to be displayed representing the option. */
     label: string,
-    /* Type is used by the menu to determine whether this is an option or a group.
-    In the case of option this is always `option`. */
+    /** Type is used by the menu to determine whether this is an option or a group.
+    In the case of option this is always `option`. **/
```

Comment:
We should make this conform with [official JSDoc](https://jsdoc.app/about-getting-started.html).
```suggestion
    /**
     * Type is used by the menu to determine whether this is an option or a group.
     * In the case of option this is always `option`.
     */
```